improve usability when mix start with no loaded project.

yann300 opened the pull request:
https://github.com/ethereum/webthree-umbrella/issues/105

yann300 committed:
- improve usability when mix start with no loaded project.

yann300 committed:
usability changes

LianaHus commented on qml/Watchers.qml:
```diff
@@ -102,7 +107,7 @@ Column {
 	}
 
-	RowLayout
+		 RowLayout
```
wrong indent :)

yann300 committed:
- indent

winsvega commented:
so now it is default project always loaded when starting mix?

Menu sections Scenario and Debug still contains active elements. 
Though now this elements not responding when no project loaded it is still active elements. (should be disabled)

yann300 commented:
I've done only a few things from your comments... @LianaHus is taking care of the rest.

yann300 committed:
Merge remote-tracking branch 'up/develop' into mix_ui

LianaHus commented on qml/StateDialog.qml:
```diff
@@ -321,6 +323,7 @@ Dialog {
 										{
 											stateAccounts.splice(styleData.row, 1)
```
we should also prevent delete of last account

LianaHus commented on qml/LogsPane.qml:
```diff
@@ -298,6 +303,14 @@ Rectangle
+				text: qsTr("Filter:")
+				verticalAlignment: Text.AlignVCenter
+				width: 50
+			}
```
"Filter" label looks like another button the user can click. We could add more info to tooltip. and I think we should make the background color a little bit more darker for toggled state

LianaHus commented on qml/StatusPane.qml:
```diff
@@ -60,19 +53,31 @@ Rectangle {
 	{
 		status.state = "";
-		status.text = "";
+		status.text = "Empty";
```
I can also suggest <no logs>

yann300 committed:
- remove Filter label
 - darker button (selected mode)
 - bugFix New Account / avoid deleting all users

yann300 committed:
bug fix
